loaders/src/OBJ/babylon.objFileLoader.ts

```diff
@@ -121,7 +121,14 @@ module BABYLON {
                 } else if (key === "map_d" && material) {
                     // The dissolve of the material
                     material.opacityTexture = MTLFileLoader._getTexture(rootUrl, value, scene);
-
+                } else if (key == 'bjs_diffuseHasAlpha' && ((material) && material.diffuseTexture)){
```

Comment:
So you want to go with a custom parameter? This does not seem scalable

Reply:
Let me review the mtl file structure and and see if there is a parameter that covers the diffuse having an alpha.  I'm pretty sure there is not, so my thought would be you could edit your mtl after export.

What thoughts do you have?

Reply:
I get your point but this is kind of not convenient
I would perhaps prefer loading the texture and trying to check if there are alpha data

But your solution is also acceptable

Reply:
roger that! Let me whip up a method for that, then maybe we can compare and then get input from others?